tests/system_tests/test_sage2lib.py

```diff
@@ -0,0 +1,29 @@
+"""Test building a dlib"""
+import sqlite3
+import subprocess
+
+import pytest
+
+
+@pytest.mark.parametrize("ftype", ["parquet", "csv"])
```

Comment:
Note: Since the whole XZ situation ive become a bit more paranoid ... we whould make a PR so the .parquet is generated from a .csv ... just to have no binaries in source control.